pkg/specgen/generate/container.go

```diff
@@ -191,9 +191,6 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 	if len(s.User) == 0 && inspectData != nil {
 		s.User = inspectData.Config.User
 	}
-	if err := finishThrottleDevices(s); err != nil {
```

Comment:
Why was this removed?

Reply:
Rather, *moved* - I see it in `container_create`.

Reply:
I think this might break things - when it was here, it ran on the host system, whereas now it runs on the server system (in APIv2 podman-remote scenarios).

Reply:
Then again, that might be a desirable change for OS X...

Reply:
I had to move is slightly down the call stack because when adding a container to a pod, the specgen doesn't get the pod's devices until `container_create` so `ResourceLimits.BlockIO.ThrottleReadBpsDevice` is filled too early